src/containers/Masthead/MastheadSearchV2.tsx

```diff
@@ -179,15 +185,30 @@ const getActiveSubjectUrl = (id: string, query: string): string => {
   return `/search?${searchParams}`;
 };
 
-interface Props {
-  root?: GQLMastheadDrawer_RootFragment;
-}
-const MastheadSearch = ({ root }: Props) => {
+const currentContextQueryDef = gql`
+  query currentContext($contextId: String!) {
+    root: node(contextId: $contextId) {
+      id
+      nodeType
+      name
+      context {
+        contextId
```

Comment:
Skal ikkje berre bruke rootId for å få tak i subject?
```suggestion
        contextId
        rootId
```

Reply:
Vi må vel ha navn uansett, må vi ikke?

Reply:
Legg til root også så får du navnet.